css-text/Overview.bs

```diff
@@ -1324,6 +1327,11 @@ Canonical order: N/A
       Shaping characters are still shaped as if the word were not
       broken, and grapheme clusters must stay together as one unit.
       No hyphenation character is inserted at the break point.
+    <dt><dfn>break-spaces</dfn></dt>
+    <dd>Line breaks are handled identically to ''overflow-wrap/normal'',
+      except that any sequence of <a>preserved</a> white space
+      that would otherwise overflow the line under these rules
+      must be broken after the last white space character that would fit the line.
```

Comment:
As a member, I still don't understand why one would want this, but as an editor, I can review PRs according to resolutions. That makes me a good reviewer though. Since I don't understand how it's used, I can review how the text reads to implementors without any background ;-)

What will happen if there are no spaces that can fit?